patroni/__init__.py

```diff
@@ -71,7 +71,7 @@ def nosync(self):
     def reload_config(self, sighup=False, local=False):
         try:
             super(Patroni, self).reload_config(sighup, local)
-            if local:
+            if local or self.api.reload_local_certificate():
```

Comment:
This condition should apply only to `self.api.reload_config(self.config['restapi'])`.
The remaining two lines should still be executed only `if local:`

Reply:
That's right, sorry for this dumb thing